src/content/docs/guides/first-command.mdx

```diff
@@ -38,7 +38,7 @@ export default class PingCommand extends Command {
 }
 ```
 
-To test if your command works correctly, you must first publish it on Discord after making minor changes to your main file:
+To test if your command works correctly, you can first publish it on Discord after making minor changes to your main file:
```

Comment:
I think for the context here "must" is right since it is a thing that user needs to do if he wants to test bot commands

Reply:
as you are refering to a test os something you can do to check if you have done right can could fit in. Must is for strong recomendations or obligation. As in the previous comment both could fit in but in my opinion can could be the best choice

Reply:
aca dejalo en "must" porque por lo que tengo entendido "can" es mas como de algo que **podrias** hacer pero must es algo como dijiste mas fuerte y llegando a ser obligatorio